docs/search/patterns.md

```diff
@@ -0,0 +1,581 @@
+---
+sidebar_position: 40
+title: Search Patterns
+---
+
+In today's world, a fundamental requirement for running a successful business is to be able to quickly and accurately locate certain information from large data repositories. This is called _indexing_ and _searching_.
```

Comment:
At a later point, let's go through and remove this marketing language. It doesn't belong in our docs.

Reply:
Please make a ticket for that and for rewriting this so that it is generic. Great content, but the presentation needs work. Might need to make a Search docs epic to hold the tickets. Don't forget to add the ImportTechDebt label when you do.

Reply:
Added a note about marketing language to DOC-393